src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -69,6 +69,8 @@ private static boolean commonHelp(String mode, boolean canConnectToRemote) {
         System.out.println("    --exe <executable for corefile>");
         if (canConnectToRemote) {
             System.out.println("    --connect [<serverid>@]<host>[:registryport][/servername] To connect to a remote debug server (debugd).");
+            System.out.println();
+            System.out.println("WARNING: debugd (and therefore --connect) are deprecated and will be removed in a future release.");
```

Comment:
This is -help output so I assume should be "The --connect option is deprecated and will be removed in a future release".